valkey.conf

```diff
@@ -2535,3 +2535,22 @@ jemalloc-bg-thread yes
 # the empty string.
 #
 # availability-zone "zone-name"
+
+################################## REPLICATION FLOW CONTROL ##################################
```

Comment:
I don't think we want a dedicated section for this, I would move it under the normal replication configs.